app/src/main/java/com/example/android/quakereport/EarthquakeActivity.java

```diff
@@ -182,12 +182,13 @@ public void onLoadFinished(Loader<List<Earthquake>> loader, List<Earthquake> ear
         mEmptyStateTextView.setText(R.string.no_earthquakes);
 
         // Clear the adapter of previous earthquake data
-        mAdapter.clear();
-
+        //mAdapter.clear();
+        
         // If there is a valid list of {@link Earthquake}s, then add them to the adapter's
         // data set. This will trigger the ListView to update.
         if (earthquakes != null && !earthquakes.isEmpty()) {
-            mAdapter.addAll(earthquakes);
+            //mAdapter.addAll(earthquakes);
+            updateUi(earthquakes);
```

Comment:
i can not able to solve updateUi(earthquakes); error  that says .... Cannot resolve method 'updateUi(java.util.List<com.example.android.quakereport.Earthquake>)'

and android studio says create method for updateUi .. how and what to pass ??

Reply:
You are calling the `updateUi` method which doesn't exists in your code yet. Try to create the same and resolve the error.

Reply:
use this code: 
mAdapter.addAll(earthquakes);
            //updateUi(earthquakes);